### DISCUSSION TITLE
I clicked something on my keyboard and now the website address in the search bar is aligned to the right, how do I make it align to the left again?

### DISCUSSION THREAD

**Discussion by KwiatekMiki** [zen-browser/desktop#1107] [Q&A]
![image](https://github.com/user-attachments/assets/fdbde94d-3431-45e9-9b41-c0716d3f1d7f)

I tried searching for a setting to do this but I can't find it.

**Comment by mr-cheffy** [comment]
https://www.google.com/search?q=urlbar+text+is+aligned+to+the+right+firefox&sourceid=chrome&ie=UTF-8

`Shift+Ctrl+X`

**Reply by KwiatekMiki** [reply]
tysm
